src/mscorlib/mscorlib.shared.sources.props

```diff
@@ -753,6 +755,29 @@
     <ThreadingSources Include="$(BclSourcesRoot)\System\Threading\Tasks\TaskToApm.cs" />
     <ThreadingSources Condition="'$(FeatureCominterop)' == 'true'" Include="$(BclSourcesRoot)\System\Threading\Tasks\IAsyncCausalityTracerStatics.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(FeatureCoreFxFileStream)' != 'true'">
+    <FileStreamSources Include="$(BclSourcesRoot)\System\IO\FileStream.cs" />
+    <SafehandleSources Include="$(BclSourcesRoot)\Microsoft\Win32\SafeHandles\SafeFileHandle.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(FeatureCoreFxFileStream)' == 'true'">
+    <FileStreamSources Include="$(CoreFxSourcesRoot)\System\IO\FileStream.cs" />
+    <FileStreamSources Include="$(CoreFxSourcesRoot)\System\IO\FileStream.NetStandard17.cs" />
+    <FileStreamSources Include="$(CoreFxSourcesRoot)\System\IO\Error.cs" />
+    <FileStreamSources Include="$(CoreFxSourcesRoot)\System\IO\StreamHelpers.ArrayPoolCopy.cs" />
+    <SafehandleSources Condition="'$(TargetsUnix)' == 'true'" Include="$(CoreFxSourcesRoot)\Microsoft\Win32\SafeHandles\SafeFileHandle.Unix.cs" />
+    <SafehandleSources Condition="'$(TargetsUnix)' != 'true'" Include="$(CoreFxSourcesRoot)\Microsoft\Win32\SafeHandles\SafeFileHandle.Windows.cs" />
+    <FileStreamSources Condition="'$(TargetsUnix)' == 'true'" Include="$(CoreFxSourcesRoot)\System\IO\FileStream.Unix.cs" />
+    <FileStreamSources Condition="'$(TargetsUnix)' != 'true'" Include="$(CoreFxSourcesRoot)\System\IO\FileStream.Win32.cs" />
+    <FileStreamSources Condition="'$(TargetsUnix)' != 'true'" Include="$(CoreFxSourcesRoot)\System\IO\FileStreamCompletionSource.Win32.cs" />
+    <FileStreamSources Condition="'$(TargetsUnix)' != 'true'" Include="$(CoreFxSourcesRoot)\System\IO\Win32Marshal.cs" />
```

Comment:
It may be useful to keep the structure same as what is in CoreFX - this file is in Common in CoreFX.